tox.ini

```diff
@@ -115,4 +115,4 @@ include_trailing_comma = True
 force_grid_wrap = 0
 line_length = 99
 known_first_party = tox
-known_third_party = pkg_resources,pluggy,py,pytest,setuptools,six
```

Comment:
@asottile I think this is a false positive of the isort plugin

Reply:
yeah, it assumes a specific layout.  PRs welcome, the missing bits are to set the application directories:

For example, here's how `reorder_python_imports` handles this:
https://github.com/asottile/reorder_python_imports/blob/794f0498340ab3adbef443431993f8a8b2046da7/reorder_python_imports.py#L405-L411

You'll want:

```yaml
    args: [--application-directories, src]
```

once `seed-isort-config` supports that

Reply:
I'm caught up now with other stuff, so if you can fix it that's cool :+1: